resources/postgres-setup-scripts/sql_start_scripts/ffc-pay-statement-data.sql

```diff
@@ -0,0 +1,69 @@
+TRUNCATE TABLE public.organisations RESTART IDENTITY CASCADE;
```

Comment:
As an example I will send you the SFI23 files, compare statement-data across the two